src/azul/service/hca_response_v5.py

```diff
@@ -261,7 +261,10 @@ def make_bundles(self, entry):
         ]
 
     def make_sources(self, entry):
-        return [{'sourceId': s['id'], 'sourceName': s['name']} for s in entry['sources']]
+        return [
```

Comment:
This line is also touched by two commits. Please don't rely on me to catch these things.

Reply:
Sorry! I must have put the `fixup!` on the wrong commit.